src/sysmonitor/SystemMonitor.php

```diff
@@ -48,8 +51,9 @@ public function collectStats(RequestStatsEvent $evt)
         // we won't even collect data for requests which don't hit either of these min-values, to prevent unnecessary garbage.
         $minQueries = 130;
         $minMemory = 80;
+        $minTime = 3;
 
-        if ($evt->usedQueries < $minQueries && $evt->peakMemory < $minMemory) {
+        if ($evt->usedQueries < $minQueries && $evt->peakMemory < $minMemory && $evt->requestTime < $minTime) {
```

Comment:
Sure. Any special reason why we shouldn`t ignore requests that don`t hit any of the limits?